apps/client/src/app/userId/[id]/page.tsx

```diff
@@ -0,0 +1,17 @@
+import QRCode from "react-qr-code";
+
+export default function Index({ params }: {
+    params: {id: string}
+}) {
+    return <div className='m-10'>
+        <h1 className='text-center mb-8' >My QR-code </h1>
+        <div>
+            <QRCode
+                size={256}
```

Comment:
Please check if QR-code looks good with those `size` and `viewBox` props on small size screens. If it's fine, then leave as it is. If it's not looking good, we should find a way to make it look good.

Reply:
It lookes good. The code is responcive. No need to correct the code.